iknow_cache.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "~> 5.0.1"
+  s.add_dependency "rails", "~> 5.1.0"
```

Comment:
Yesterday we were talking about having broader requirements such as `>=5.0 && <5.2` - did we want that on all our gems?